diff --git a/rlinf/envs/__init__.py b/rlinf/envs/__init__.py
--- a/rlinf/envs/__init__.py
+++ b/rlinf/envs/__init__.py
@@ -37,6 +37,7 @@ class SupportedEnvType(Enum):
     HABITAT = "habitat"
     OPENSORAWM = "opensora_wm"
     WANWM = "wan_wm"
+    OPENDW_ROBOTWIN = "opendw_robotwin"
     GENESIS = "genesis"
     EMBODICHAIN = "embodichain"
     ROBOVERSE = "roboverse"
@@ -139,6 +140,10 @@ def get_env_cls(env_type: str, env_cfg=None):
         from rlinf.envs.world_model.world_model_wan_env import WanEnv
 
         return WanEnv
+    elif env_type == SupportedEnvType.OPENDW_ROBOTWIN:
+        from rlinf.envs.world_model.opendw_robotwin_env import OpenDWRobotwinEnv
+
+        return OpenDWRobotwinEnv
     elif env_type == SupportedEnvType.EMBODICHAIN:
         from rlinf.envs.embodichain.embodichain_env import EmbodiChainEnv
 
